### ISSUE TITLE
ExplanationDashboard goes blank on opening what if tab

### ISSUE THREAD

**Issue by shivamsingha** [microsoft/responsible-ai-widgets#880]
ExplanationDashboard goes blank on opening what if tab.
[Here's a screencapture of what's happening.](https://www.youtube.com/watch?v=i0lA9OajXlg)

React throws this error `Uncaught TypeError: predictedProb is undefined`
```
Uncaught TypeError: predictedProb is undefined
    render http://localhost:5000/:422546
    finishClassComponent http://localhost:5000/:369356
    updateClassComponent http://localhost:5000/:369306
    beginWork http://localhost:5000/:370944
    callCallback http://localhost:5000/:355816
    invokeGuardedCallbackDev http://localhost:5000/:355865
    invokeGuardedCallback http://localhost:5000/:355927
    beginWork$1 http://localhost:5000/:375830
    performUnitOfWork http://localhost:5000/:374642
    workLoopSync http://localhost:5000/:374573
    renderRootSync http://localhost:5000/:374536
    performSyncWorkOnRoot http://localhost:5000/:374159
    flushSyncCallbackQueueImpl http://localhost:5000/:363198
    unstable_runWithPriority http://localhost:5000/:383173
    runWithPriority$1 http://localhost:5000/:363147
    flushSyncCallbackQueueImpl http://localhost:5000/:363193
    flushSyncCallbackQueue http://localhost:5000/:363180
    discreteUpdates$1 http://localhost:5000/:374286
    discreteUpdates http://localhost:5000/:355627
    dispatchDiscreteEvent http://localhost:5000/:357760
localhost:5000:422546:17
Warning: Using UNSAFE_componentWillReceiveProps in strict mode is not recommended and may indicate bugs in your code. See https://reactjs.org/link/unsafe-component-lifecycles for details.

* Move data fetching code or side effects to componentDidUpdate.
* If you're updating state whenever props change, refactor your code to use memoization techniques or move it to static getDerivedStateFromProps. Learn more at: https://reactjs.org/link/derived-state

Please update the following components: Autofill, ComboBox, DropdownBase, SearchBoxBase react_devtools_backend.js:4049:25
The above error occurred in the <ExistingPredictionLabels> component:

ExistingPredictionLabels@http://localhost:5000/:422344:1
div
ResultComponent@http://localhost:5000/:13324:99
div
ResultComponent@http://localhost:5000/:13324:99
div
ResultComponent@http://localhost:5000/:13324:99
WhatIfPanel@http://localhost:5000/:423389:5
div
div
WhatIfTab@http://localhost:5000/:423985:5
div
ResultComponent@http://localhost:5000/:13324:99
div
ResultComponent@http://localhost:5000/:13324:99
div
NewExplanationDashboard@http://localhost:5000/:426183:5
Interpret@http://localhost:5000/:432210:1
App@http://localhost:5000/:431980:1

Consider adding an error boundary to your tree to customize error handling behavior.
Visit https://reactjs.org/link/error-boundaries to learn more about error boundaries. react_devtools_backend.js:4049:25
```

```
print(raiwidgets.version)
0.10.0
```

**Comment by imatiach-msft** [comment]
@shivamsingha  thank you for finding this, it seems to be a regression caused by the recent PR #652 .  I've sent a PR to fix this issue here:
https://github.com/microsoft/responsible-ai-widgets/pull/887

**Comment by imatiach-msft** [comment]
PR has been merged and the fix is in raiwidget 0.11.0 which was just released to pypi, so closing this issue

**Comment by shivamsingha** [comment]
This problem still exists on the new version.
